now another test failed 🤔
```
FAILED TESTS:
11:07:27       selectStatements
11:07:27         ✖ coalesces statements from different apps
11:07:27           Chrome Headless 96.0.4664.45 (Linux x86_64)
11:07:27         AssertionError: expected 2 to equal 1
11:07:27             at Context.eval (webpack-internal:///./src/views/statements/statements.spec.tsx:83:49)
```

--- a/pkg/ui/workspaces/db-console/src/redux/queryManager/saga.spec.ts
+++ b/pkg/ui/workspaces/db-console/src/redux/queryManager/saga.spec.ts
@@ -143,7 +143,7 @@ describe("Query Management Saga", function() {
         let queryCalled = 0;
         const selfStopQuery = {
           id: "selfStopQuery",
-          refreshInterval: moment.duration(50),
+          refreshInterval: moment.duration(5),
           querySaga: function*(): IterableIterator<void> {
             queryCalled++;
             if (queryCalled > 3) {
@@ -156,9 +156,9 @@ describe("Query Management Saga", function() {
           .dispatch(autoRefresh(selfStopQuery))
           .dispatch(autoRefresh(selfStopQuery))
           .dispatch(autoRefresh(selfStopQuery))
-          .silentRun(250)
+          .silentRun()
           .then(() => {
-            assert.equal(queryCalled, 5);
+            assert.equal(queryCalled, 6);
           });
       });
       it("Uses retry delay when errors are encountered", function() {

